src/api/client.js

```diff
@@ -12,7 +12,20 @@ export function pingStudioHome() {
   );
 }
 
-export function getAssets(courseId, { page = 0, pageSize = 50, sort = 'sort', assetType = '' }) {
+function assetTypeValueFromState(assetFilters) {
+  // TODO: change this to multi-value once API supports it
```

Comment:
@MichaelRoytman once your API PR merges, this function can be updated to output a comma-separated string instead of just one asset_type.